pkg/apis/aws/types_infrastructure.go

```diff
@@ -87,6 +87,8 @@ type VPC struct {
 	ID *string
 	// CIDR is the VPC CIDR.
 	CIDR *string
+	// GatewayEndpoints service names to configure as gateway endpoints in the VPC.
+	GatewayEndpoints []string
```

Comment:
You have to add this to `pkg/apis/aws/v1alpha1/types_infrastructure.go` and run `make generate`
Also, please add `// +optional` and `omitempty` tags there.

Reply:
done, still failing

Reply:
you have not run `make generate`